app/adminPage.html

```diff
@@ -91,6 +93,17 @@ <h4 class="search-heading">Filter Results</h4>
                     <tbody class="score_list user-table"></tbody>
                 </table>
             </div>
+            <form class="new-test-container" id="testForm" method="post">
+                <h2>Add new test</h2>
+                <input name="testName" type="text" id="testName" placeholder="Test name">
+                <input type="submit" class="btn primary-button new-test-button submit" value="Add new test">
+                <div id="message-target"></div>
+                <div class="buttons">
+                    <div class="messageArea">
```

Comment:
it might be good to use the bootstrap alerts here for your success and failure messages, they are nicely styled and can be dismissed. look here:
https://getbootstrap.com/docs/4.0/components/alerts/

Reply:
done :)